tests/e2e/performance/test_pvc_multi_clone_performance.py

```diff
@@ -0,0 +1,388 @@
+
+        Args:
+            cmd (str/list): the command to execute
+
+        Returns:
+            list : all STDOUT / STDERR output as list of lines
+
+        """
+        if isinstance(cmd, str):
+            command = cmd.split()
+        elif isinstance(cmd, list):
+            command = cmd
+        else:
+            return self.params['ERRMSG']
+
+        log.info(f'Going to run {cmd}')
+        cp = subprocess.run(
+            command,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            stdin=subprocess.PIPE,
+            timeout=600
+        )
+        output = cp.stdout.decode()
+        err = cp.stderr.decode()
+        # exit code is not zero
+        if cp.returncode:
+            log.error(f'Command finished with non zero ({cp.returncode}) {err}')
+            output += f"{self.params['ERRMSG']} {err}"
+
+        output = output.split('\n')  # convert output to list
+        output.pop()  # remove last empty element from the list
+        return output
+
+    def run_oc_command(self, cmd, namespace):
+        """
+        Running an 'oc' command
+
+        Args:
+            cmd (str): the command to run
+            namespace (str): the namespace where to run the command
+
+        Returns:
+            list : the results of the command as list of lines
+
+        """
+
+        cluster_dir_kubeconfig = os.path.join(
+            config.ENV_DATA['cluster_path'],
+            config.RUN.get('kubeconfig_location')
+        )
+        if os.getenv('KUBECONFIG'):
+            kubeconfig = os.getenv('KUBECONFIG')
+        elif os.path.exists(cluster_dir_kubeconfig):
+            kubeconfig = cluster_dir_kubeconfig
+        else:
+            kubeconfig = None
+
+        command = (f'oc --kubeconfig {kubeconfig} -n {namespace} {cmd}')
+        return self.run_command(command)
+
+    def measure_clone_creation_time(self, clone_name, start_time):
+        """
+
+
+        Args:
+
+        Returns:
+
+
+        """
+        logs = []
+        for l in self.params['logs']:
+            logs.append(
+                self.run_oc_command(
+                    f"logs {l} -c csi-provisioner --since-time={start_time}",
+                    "openshift-storage",
+                )
+            )
+
+        format = "%H:%M:%S.%f"
+
+        st = None
+        et = None
+        for sublog in logs:
+            for line in sublog:
+                if st is None and "provision" in line and clone_name in line and "started" in line:
+                    st = line.split(" ")[1]
+                    st = datetime.datetime.strptime(st, format)
+                elif et is None and "provision" in line and clone_name in line and "succeeded" in line:
+                    et = line.split(" ")[1]
+                    et = datetime.datetime.strptime(et, format)
+                if (st is not None and et is not None):
+                    break;
+            if (st is not None and et is not None):
+                break;
```

Comment:
Remove semicolon

Reply:
Done, will be pushede in the next commit. Thank you!